doc/api/module.md

```diff
@@ -399,6 +399,28 @@ the [`NODE_COMPILE_CACHE=dir`][] environment variable if it's set, or defaults
 to `path.join(os.tmpdir(), 'node-compile-cache')` otherwise. To locate the compile cache
 directory used by a running Node.js instance, use [`module.getCompileCacheDir()`][].
 
+By default, cache keys are computed using the absolute paths of modules.
+This means the cache is not reusable if the project directory is moved or
+copied elsewhere.
+To make the cache portable, relative path computation can be enabled for
+compile cache. This allows previously compiled modules to be reused across
+different directory locations as long as the relative layout remains the same.
+
+There are two ways to enable the portable mode:
+Using the portable option in module.enableCompileCache():
+
+```js
+// Absolute paths (default): cache breaks if project is moved
+module.enableCompileCache({ path: '.cache' });
+
+// Relative paths (portable): cache works after moving project
+module.enableCompileCache({ path: '.cache', portable: true });
+```
```

Comment:
```suggestion
1. Using the portable option in `module.enableCompileCache()`:
  
  ```js
  // Absolute paths (default): cache breaks if project is moved
  module.enableCompileCache({ path: '.cache' });
  
  // Relative paths (portable): cache works after moving project
  module.enableCompileCache({ path: '.cache', portable: true });
  ```
2. Or by setting the environment variable: [`NODE_COMPILE_CACHE_PORTABLE=1`][]
```

(It doesn't seem to render correctly on GitHub, I am just trying to suggest to move the block into the list.